openreview/profile/process/dblp_abstract_process.js

```diff
@@ -0,0 +1,28 @@
+async function process(client, edit, invitation) {
```

Comment:
why the abstract invitation should have a process function to extract the abstract?

This is invitation is being to set the abstract value

Reply:
to allow manual triggering of abstract extraction when the process function in DBLP.org/-/Record fail

Reply:
but the abstract in the edit would be empty or what?

if the edit has a value, do you overwrite it?

and it is seems to be an infinite loop here, you are posting an edit with the same invitation of the process function.